app/components/move-drag-button.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box, BoxProps } from "@mui/material";
+
+import { Icon } from "@/icons";
+
+type MoveDragButtonProps = {
+  className?: string;
+  dragButtonProps?: BoxProps;
```

Comment:
I would not have an inner dragButtonProps, but have an icon className; since the MoveDragButton is the container, I expect props to be passed to the parent component.